src/components/generate/index.js

```diff
@@ -0,0 +1,116 @@
+import { useState, useCallback, Fragment, useEffect } from 'react';
+import Input from '../input';
+import Select from '../select';
+import Button from '../button';
+import SnackBar from '../snackbar';
+import Loader from '../loader';
+import Error from '../error';
+import classes from './styles.module.css';
+import { isEmptyString, copyText } from '../../utils';
+import { SEVERITY } from '../constants';
+
+const Generate = () => {
+    const [inputTextVal, setInputTextVal] = useState('');
+    const [inputSecretKey, setInputSecretKey] = useState('');
+    const [validity, setValidity] = useState('');
+    const [loading, setLoading] = useState(false);
+    const [openSnackbar, setOpenSnackbar] = useState(false);
+    const [errorMessage, setErrorMessage] = useState('');
+    const [errorStatus, setErrorStatus] = useState('');
+    const [snackbarMessage, setSnackbarMessage] = useState('');
+    const [snackbarSeverity, setSnackbarSeverity] = useState(SEVERITY.INFO);
+    const [fetchResult, setFetchResult] = useState(false);
+    const [url, setUrl] = useState('');
+    const [urlCopied, setUrlCopied] = useState(false);
+
+    useEffect(() => fetchResult && sendNewMessageRequest(), [fetchResult]);
```

Comment:
sendNewMessageRequest missing from dependency array

Reply:
done but need to discuss...